opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/CommonGivenStages.java

```diff
@@ -52,6 +54,8 @@
     @Autowired
     private RequestSigningService signingService;
 
+    protected Set<String> createdUsers = new HashSet<>();
```

Comment:
As this set can be called from different invocation points, it should be thread-safe, but, as it is only 1 test that requires this check - probably, we can contain this functionality inside the test itself and not extend this class

Reply:
changed to thread safe implementation. I think we can use this feature more then once